roles/ood_user_reg_ops/tasks/main.yml

```diff
@@ -0,0 +1,23 @@
+- name: Get UID and GID of RegUser from OHPC
+  stat: 
+     path: /home/{{ RegUser_app_user }}
+  register: RegUser_stat
+
+- name: Create Group for RegUser on OOD
+  group:
+    gid: "{{ RegUser_stat.stat.gid  }}"
+    name: "{{ RegUser_app_user  }}"
+    state: present
+
+- name: Create RegUser on OOD
+  user:
+    name: "{{ RegUser_app_user }}"
+    comment: "{{ RegUser_app_user_full_name }}"
+    shell: /bin/bash
+    uid: "{{ RegUser_stat.stat.uid }}"
+    group: "{{ RegUser_app_user  }}"
+    createhome: no
+    state: present
+
+- name: restart httpdd
+    service: name=httpd state=restarted
```

Comment:
service name should be `httpd24-httpd` instead of `httpd`. The indent here is wrong